packages/flutter/lib/src/cupertino/list_section.dart

```diff
@@ -0,0 +1,814 @@
+  darkElevatedColor: Color.fromRGBO(142, 142, 146, 1.0),
+  highContrastElevatedColor: Color.fromRGBO(108, 108, 108, 1.0),
+  darkHighContrastElevatedColor: Color.fromRGBO(142, 142, 146, 1.0),
+);
+
+enum _CupertinoListSectionType { base, insetGrouped }
+
+/// An iOS-style list section.
+///
+/// The [CupertinoListSection] is a container for children widgets. These are
+/// most often [CupertinoListTile]s.
+///
+/// The base constructor for [CupertinoListSection] constructs an
+/// edge-to-edge style section which includes an iOS-style header, the dividers
+/// between rows, and borders on top and bottom of the rows. An example of such
+/// list section are sections in iOS Settings app.
+///
+/// The [CupertinoListSection.insetGrouped] constructor creates a round-edged
+/// and padded section that is seen in iOS Notes and Reminders apps. It creates
+/// an iOS-style header, and the dividers between rows. Does not create borders
+/// on top and bottom of the rows.
+///
+/// The section [header] lies above the [children] rows, with margins and style
+/// that match the iOS style.
+///
+/// The section [footer] lies below the [children] rows and is used to provide
+/// additional information for current list section.
+///
+/// The [children] is the list of widgets to be displayed in this list section.
+/// Typically, the children are of type [CupertinoListTile], however these is
+/// not enforced.
+///
+/// The [margin] is used to provide spacing around the content area of the
+/// section encapsulating [children].
+///
+/// The [decoration] of [children] specifies how they should be decorated. If it
+/// is not provided in constructor, the background color of [children] defaults
+/// to [CupertinoColors.secondarySystemGroupedBackground] and border radius of
+/// children group defaults to 10.0 circular radius when constructing with
+/// [CupertinoListSection.insetGrouped]. Defaults to zero radius for the
+/// standard [CupertinoListSection] constructor.
+///
+/// The [dividerMargin] and [additionalDividerMargin] specify the starting
+/// margin of the divider between list tiles. The [dividerMargin] is always
+/// present, but [additionalDividerMargin] is only added to the [dividerMargin]
+/// if `hasLeading` is set to true in the constructor, which is the default
+/// value.
+///
+/// The [backgroundColor] of the section defaults to
+/// [CupertinoColors.systemGroupedBackground].
+///
+/// {@macro flutter.material.Material.clipBehavior}
+///
+/// {@tool snippet}
+/// Creates a base [CupertinoListSection] containing [CupertinoListTile]s with
+/// `leading`, `title`, `additionalInfo` and `trailing` widgets.
+///
+/// ```dart
+/// class MyStatefulWidget extends StatefulWidget {
+///   const MyStatefulWidget({Key? key}) : super(key: key);
+///
+///   @override
+///   _MyStatefulWidgetState createState() => _MyStatefulWidgetState();
+/// }
+///
+/// class _MyStatefulWidgetState extends State<MyStatefulWidget> {
```

Comment:
**chore**: Since your state class is just overriding `build` and nothing more, make the example with a `StatelessWidget`

Reply:
Fixed.